plugins/shared/info-panel/error.handlebars

```diff
@@ -11,5 +11,10 @@
             {{{title}}}
         </div>
     </a>
-    <div class="tota11y-info-error-description tota11y-collapsed"></div>
+    <div class="tota11y-info-error-description tota11y-collapsed">
+        <div class="tota11y-info-error-description-code"> 
+            Surrounding code:
+            <code></code> 
```

Comment:
This is where our class names haunt us, they get super long :)

How about we name the container `.tota11y-info-error-description-code-container` and give *this* one `.tota11y-info-error-description-code`? And reflect those changes in the Less.